lib/cocina/models/validators/description_values_validator.rb

```diff
@@ -62,22 +62,30 @@ def validate_values_for_blanks(hash, path)
         end
 
         def validate_values_for_multiples(hash, path)
-          return unless hash.count { |key, value| %i[value groupedValue structuredValue parallelValue].include?(key) && value.present? } > 1
+          return unless hash.count { |key, value| %w[value groupedValue structuredValue parallelValue].include?(key) && value.present? } > 1
```

Comment:
keys are strings, rather than symbols.

Reply:
How do we know these keys are strings?

The entrypoint to the `DescriptionValuesValidator` class is the `Validator` class, which [converts the attributes to be validated into a `HashWithIndifferentAccess` instance](https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/models/validators/validator.rb#L24-L25).

The `HashWithIndifferentAccess` class maps symbol keys to string keys in its writing interface. See https://api.rubyonrails.org/classes/ActiveSupport/HashWithIndifferentAccess.html. Thus:
> You are guaranteed that the key is returned as a string: